internals/overlord/logstate/opentelemetry/opentelemetry_test.go

```diff
@@ -83,117 +83,97 @@ func (*suite) TestRequest(c *C) {
 {"resourceLogs": [
 	{
 		"resource": {
-			"attributes": [{"key": "service.name", "value": {"stringValue": "svc1"}}]
+			"attributes": null
```

Comment:
Why did this change to null? Shouldn't service.name always be there?

Reply:
Because in the previous comment, you mentioned that since we already have a dedicated test that tests `SetLabels`, I removed it from here, and the default label `service.name` is set in `SetLabels`, same as the Loki package. In `gatherer`, this won't be a problem, since when the plan is changed, it always sets labels.